packages/ses/README.md

```diff
@@ -647,17 +646,16 @@ buffer to construct a high resolution timer.
 
 The `ses` shim does not in itself isolate the stack of guest programs, even
 when evaluated in separate compartments.
-This is relevant when program created objects are shared between guest
-programs.
+This is relevant when objects are shared between guest programs.
 
 When a program interacts with an object introduced by another program (as
 through the per-compartment `globalThis`, function arguments or returned
 values), there are potential risks due to the synchronous nature of object
 access.
 Even interactions that are not explicit function calls may cause code from
 another program, like property accessors or proxy traps, to execute on the same
-stack, which may be able to call back into the program (reentrancy), throw, or
-detect the current stack height.
+stack, which may be able to detect the stack height, throw an exception, or call
+back into the program in pursuit of a reentrancy attack.
```

Comment:
Why do we use “reentrancy” when “reentrance” is right there? (I know why we don’t use “reënter”, mwahah)